Mathlib/CategoryTheory/Galois/Prorepresentability.lean

```diff
@@ -0,0 +1,235 @@
+/-
+Copyright (c) 2024 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.Algebra.Category.GroupCat.Basic
+import Mathlib.CategoryTheory.Galois.Decomposition
+import Mathlib.CategoryTheory.Limits.FunctorCategory
+
+/-!
+# Pro-Representability of fiber functors
+
+We show that any fiber functor is pro-representable, i.e. there exists a pro-object
+`X : I ⥤ C` such that `F` is naturally isomorphic to `X ⋙ coyoneda`.
```

Comment:
Do not you mean the *colimit* of `X ⋙ coyoneda`?

Reply:
Indeed, that's what I mean.